### DISCUSSION TITLE
Is it possible to get some Test Elements into Hammer?

### DISCUSSION THREAD

**Discussion by SparkleArts** [BEEmod/BEE2-items#4280] [General]
I wanted to know if is it possible to get the Test Elements as a Instance into Hammer.

I prefer Hammer for the maps and would like to test a few test elements in Hammer.

**Comment by ItzAlmaZ** [comment]
If I understand the question correctly
Maps with Beemod items can be opened, but they will not be exactly the same as in a regular editor. The pipe will not be a pipe, but some kind of trigger.

**Comment by slambo311** [comment]
They're all already instances, that's how PeTI works.
